### PR TITLE
Bump compose bom

### PR DESCRIPTION
**Pull request by MV-GH**

### CODE REVIEW

**Review by dessalines — APPROVED** [review]
Thx. Surprised that renovate didn't pick this one up.

Edit: actually nm, its not finding that deployed yet.

**Comment by MV-GH** [comment]
Strange it has been released a week ago

https://github.com/chrisbanes/compose-bom/releases

File: app/build.gradle.kts [hunk 1]
```diff
@@ -112,7 +112,7 @@ dependencies {
     // https://github.com/LemmyNet/jerboa/pull/1502#issuecomment-2137935525
     // val composeBom = platform("androidx.compose:compose-bom:2024.05.00")
 
-    val composeBom = platform("dev.chrisbanes.compose:compose-bom:2024.05.00-alpha03")
+    val composeBom = platform("dev.chrisbanes.compose:compose-bom:v2024.06.00-alpha01")
```
Comment (dessalines) [inline app/build.gradle.kts:115]: I bet its because you have a `v2024`, it should be just `2024....`

Comment (dessalines) [inline app/build.gradle.kts:115]:
```suggestion
    val composeBom = platform("dev.chrisbanes.compose:compose-bom:2024.06.00-alpha01")
```

Comment (dessalines) [reply app/build.gradle.kts]: I got it.

**Pull request merged**